index.js

```diff
@@ -3,18 +3,17 @@
 const fs = require('fs-promise')
 const path = require('path')
 
+const stdin = require('get-stdin')
+const stream = require('stream')
+
 const chalk = require('chalk')
-const ora = require('ora')
-const spinner = ora()
+const spinner = require('ora')()
```

Comment:
When processing multiple files, you don't get a full listing of files processed.